.NET/Microsoft.Recognizers.Text.Number/Microsoft.Recognizers.Text.Number.xml

```diff
@@ -167,6 +167,15 @@
             <param name="extResult">input arabic number.</param>
             <returns>parsed result.</returns>
         </member>
+        <member name="M:Microsoft.Recognizers.Text.Number.BaseNumberParser.GetSplitIndex(System.Collections.Generic.List{System.String})">
+            <summary>
+            Get the split index for a fraction word list, split index used to separate the numerator and the denominator.

```

Comment:
Is this still needed in the xml now that the method is private?

Reply:
If your method has documentation comments, it will automatically add member in the xml file when build.